pyproject.toml

```diff
@@ -35,18 +35,19 @@ documentation = "https://softwaremill.github.io/FoXAI/"
 "Bug Tracker" = "https://github.com/softwaremill/foxai/issues"
 
 [tool.poetry.dependencies]
-python = ">=3.7.2,<3.11"
+python = ">=3.8.0,<3.11"
 numpy = ">=1.21.4,<2.0.0"
 captum = ">=0.5.0,<1.0.0"
-torch = ">=1.12.1,<2.0.0"
-pytorch-lightning = ">=1.5.0,<2.0.0"
+torch = ">=2.0.0,<=2.0.1"
```

Comment:
I think we can move straight away to Pytorch version 2.0 as it will be easier to support everything (for just one Pytorch), and we do not have so many legacy systems needing older Pytorch support. @adamwawrzynski @adamjankaczmarek WDYT?

Reply:
First of all, we should test the library against torch version `2.0+` and check if it works. I think we might give it a try to support a broader range of versions `>1.12.1` (without an upper threshold). If that would be impossible we could switch to `2.0+`.

I think there are plenty of libraries and code samples for older torch versions and it would be great to make it possible to use with our library.

Reply:
outdated - resolving. Currently tests show that `torch==1.21.1` can be supported.